opentreemap/treemap/js/src/lib/searchBar.js

```diff
@@ -111,6 +112,25 @@ function initSearchUi(searchStream) {
         geocoderBbox: config.instance.extent
     });
 
+    var $query_typeaheads = $('.search-right .autocomplete-group');
+    $query_typeaheads.each(function () {
+        var $textInput = $(this).find('[type="text"]');
+        var $hiddenInput = $(this).find('[type="hidden"]');
+        // Can't use # notation because id contains a dot
+        var textId = '[id="' + $textInput.attr('id') + '"]';
+        var hiddenId = '[id="' + $hiddenInput.attr('id') + '"]';
```

Comment:
Given the way these are used in `otmTypeahead.js` (immediately wrapped in `$(...)`), you don't need to do this at all, you can just pass these in directly like so:
```javascript
otmTypahead.create({
    input: $textInput,
    hidden: $hiddenInput,
    ...
});
```